docs/v0.5/getting-started/docker-for-mac.md

```diff
@@ -42,31 +42,51 @@ Watch pods in a separate terminal.
 watch -n 1 kubectl get pod --all-namespaces
 ```
 
-## Install Helm
+## Install kapp
 
-[Helm](https://helm.sh) is a popular package manager for Kubernetes. The riff runtime and its dependencies are provided as Helm charts.
+[kapp](https://get-kapp.io/) is a simple deployment tool for Kubernetes. The riff runtime and its dependencies are provided as standard Kubernetes yaml files, that can be installed with kapp.
 
-Download a recent release of the [Helm v2 CLI](https://github.com/helm/helm/releases/) for your platform.
-(Download version 2.13 or later, Helm v3 is currently pre-release and has not been tested for compatibility with riff).
-Unzip and copy the Helm CLI executable to a directory on your path.
-
-Initialize the Helm Tiller server in your cluster.
+You install kapp using Homebrew:
 ```sh
-kubectl create serviceaccount tiller -n kube-system
-kubectl create clusterrolebinding tiller --clusterrole cluster-admin --serviceaccount kube-system:tiller
-helm init --wait --service-account tiller
+brew tap k14s/tap
+brew install kapp
 ```
 
+Alternatively, Download a recent binary for your platform from [github]((https://github.com/k14s/kapp/releases)).
+Move it into a directory on your path, and make it executable.
+Complete kapp installation instructions can be found [here](https://k14s.io/#install-from-github-release)
+
 Validate the installation.
 ```sh
-helm version
+kapp version
 ```
 ```
-Client: &version.Version{SemVer:"v2.16.0", GitCommit:"618447cbf203d147601b4b9bd7f8c37a5d39fbb4", GitTreeState:"clean"}
-Server: &version.Version{SemVer:"v2.16.0", GitCommit:"618447cbf203d147601b4b9bd7f8c37a5d39fbb4", GitTreeState:"clean"}
+Client Version: 0.17.0
+
+Succeeded
 ```
 
-> NOTE: Please see the [Helm documentation](https://helm.sh/docs/using_helm/#securing-your-helm-installation) for additional Helm security configuration.
+## Install ytt
+
+[ytt](https://get-ytt.io/) is a tool for templating yaml. It can be used to apply changes to the distributed Kubernetes yamls files used to install riff.
+
+You install ytt using Homebrew:
+```sh
+brew tap k14s/tap
+brew install ytt
+```
+
+Alternatively, Download a recent binary for your platform from [github]((https://github.com/k14s/ytt/releases)).
```

Comment:
too many parentheses in the github link
![Screen Shot 2020-01-13 at 5 41 05 PM](https://user-images.githubusercontent.com/849592/72278464-f7605280-362b-11ea-9062-9a808cc349b4.png)